packages/edit-site/src/components/create-template-part-modal/style.scss

```diff
@@ -8,16 +8,6 @@
 	}
 }
 
-
-.edit-site-create-template-part-modal__modal-actions {
-	padding-top: $grid-unit-15;
-}
-
-.edit-site-create-template-part-modal__area-base-control .components-base-control__label {
-	margin: $grid-unit-20 0 $grid-unit-10;
-	cursor: auto;
```

Comment:
Is there a reason why we'd use a text cursor here?

Reply:
Wow this is interesting, I've never noticed it before. The standard WordPress forms.css adds a `cursor: pointer` to all `label`s:

<img width="252" alt="" src="https://user-images.githubusercontent.com/555336/212898431-41a09e5a-c141-49b8-be76-fbbc79618056.png">

I'm guessing the intent was to add an affordance that the labels are clickable. But unfortunately that is not always true, as is in this case (and some others, e.g. [`<progress>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress)). So that's probably why they explicitly overrode the `cursor: pointer` here.

cc @ciampo FYI